paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheet.kt

```diff
@@ -122,7 +122,15 @@ class PaymentSheet internal constructor(
          *
          * See [payment-notification](https://stripe.com/docs/payments/payment-methods#payment-notification).
          */
-        val allowsDelayedPaymentMethods: Boolean = false
+        val allowsDelayedPaymentMethods: Boolean = false,
+
+        /**
+         * PaymentSheet offers users an option to save some payment methods for later use.
+         * By default, this option is enabled, and the user can opt-out.
+         * Setting this to `true` will require that the user explicitly opt-in
+         * (e.g. by checking a box) to save their payment method.
+         */
+        val requiresUserOptInToSavePaymentMethod: Boolean = false
```

Comment:
If this is a requirement for EU merchants, is it worth adding that in the comment here?

Reply:
cc: @ahmedg-stripe